roles/ocp_on_libvirt/README.md

```diff
@@ -20,4 +45,46 @@ Please add the following variables to the libvirt cluster configuration:
     bus: "0x86"
     slot: "0x00"
     function: "0x0"
+```
+
+## How to create a libvirt cluster based on Virtualmedia over Redfish
+
+### Example of inventory file
+
+```
+all:
+  children:
+    vm_host:
+      hosts:
+        libvirthost:
+          vbmc_user: MY_USER
+          vbmc_pass: MY_PASSWORD
+    nodes:
+      hosts:
+        libvirthost
+  vars:
+    bootstrapProvisioningIP: 192.168.168.168
+    externalMACAddress: 01:23:45:67:89:ab
+    rhsm_username: !vault |
+      $ANSIBLE_VAULT;1.1;AES256
+      ### REDACTED ###
+    rhsm_password: !vault |
+      $ANSIBLE_VAULT;1.1;AES256
+      ### REDACTED ### 
+```
+
+### Example of encrypting the RHSM password
+
+```
+$ ansible-vault encrypt_string --name rhsm_password this_is_my_password
+New Vault password: 
+Confirm New Vault password: 
+Encryption successful
+rhsm_password: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          32346162626130663866346162643139663139376366316330626366396235343463666439303961
+          3634363239323265383265386132316139323061616332660a333132393335313262313636323636
+          31333139636565336333363935323536333837636537633335646134393465633962393165653837
+          3538336132316337300a323336373839643165633465333630346666623738633433653232336138
+          66656230323663613361333335656666623333326431633532633766353435373336
```

Comment:
I'm sorry, I missed these lines, I thought rhsm_* vars are not used in the role, I don't think these examples should be here. Should be them documented in the place where those are used?

Reply:
I believe I'll move this documentation to the team's internal documentation.

Reply:
Is this done? Is something more needed to merge this change?

Reply:
This question still remains to be answered

Reply:
Sorry. Haven't been paying much attention to this PR lately.
The actions are done, but now I do believe that we may need to add some documentation regarding the usage of RHEL images in the VMs.

Reply:
@nsilla  perhaps it was forgotten? But the Example to encrypt RHSM is *still* in here, this doc is suited for a different place, but not this role.